config/routes.rb

```diff
@@ -292,7 +292,11 @@
     end
 
     namespace :delivery_partners, path: "delivery-partners" do
-      resources :users, only: %i[index new create]
+      resources :users, only: %i[index new create edit update destroy] do
+        member do
+          get "delete", action: :delete
```

Comment:
why is this a `get`?

Reply:
oh ignore me i see it now, its show the delete page

Reply:
@asmega this is a confirmation page because destroy is done